lib/plugin-manager.js

```diff
@@ -90,8 +90,10 @@ class PluginManager extends EventEmitter {
       this._loadSourcesFromIndex();
     });
 
-    this.metadata = new Metadata();
-    this.metadata.service.host = configuration.metadataHost || Config.get('metadata:host');
+    this.metadata = new Metadata({
+      host: configuration.metadataHost || Config.get('metadata:host'),
+      interval: configuration.metadataHost || Config.get('metadata:interval')
```

Comment:
Looks like a copy paste error? I think this should be `configuration.metadataInterval`.